fog/sample-paykit/src/cached_tx_data/memo_handler.rs

```diff
@@ -156,6 +144,18 @@ impl MemoHandler {
                     Err(MemoHandlerError::FailedSubaddressValidation)
                 }
             }
+            MemoType::GiftCodeCancellation(_) => {
```

Comment:
It looks like this is just moving code around, why is it being moved?

Reply:
Oh, I had moved this code around to group the memo types together when I had previously added the "withData" types. And then I just left these like this because I figured it was cleaner than before.

Reply:
I would discourage making unnecessary changes like that, it adds noise to the diff and can confuse reviewers.

One thing i like to do is call out explicitly with a PR comment "this is just code movement, no functional change", because when I read a diff and don't see the change, i assume i missed it and read it again

Reply:
That makes sense. Do you think this discussion serves that purpose, or should I make a separate PR comment to call it out?